### PR TITLE
Feature: added typography mixins and Montserrat font configuration

### PR DESCRIPTION
**Pull request by biryukov-m**
### Updates:

**Typography SCSS Mixins:**
Added new SCSS mixins for typography to ensure consistent styling across the project. 
The mixins include:
- text-h1 for headers with a font size of 48px. (equal rem)
- text-h2 for headers with a font size of 40px.
- text-subtitle for subtitles with a font size of 28px.
- text-card-title for card titles with a font size of 24px.
- text-accent for accent text with a font size of 20px.
and others.
All mixins use letter-spacing: 0.01em, which corresponds to 1% of the font size.

**Font Configuration:**
Configured the Montserrat font using next/font/google. Added support for different font weights (Regular, Medium, SemiBold) and language subsets (Latin, Cyrillic, Latin-ext) for English, Polish, and Ukrainian.

**Breakpoint variables:**
Add breakpoint vars according to design

**Application on Main Page:**
Applied the new typography mixins to the main page to demonstrate the usage and ensure the styles are rendered as expected.

**Changes Made:**
Updated styles/mixins.scss with new typography mixins.
Configured Montserrat font in src/app/layout.tsx.
Applied typography styles to the main page components.
Removed template styles in global.scss

**Testing:**
Verified that typography styles are correctly applied to all relevant components.
Checked that Montserrat font renders correctly with the specified weights and subsets.

### CODE REVIEW

File: src/app/Page.module.scss
Comment (OlhaPo) [inline src/app/Page.module.scss:1]: Maybe such class names like one, two, three etc might be confusing, better to use more descriptive and meaningful names. My suggestion: heading-primary, heading-secondary, subtitle, card-title, accent-text, main-text, big-btn-text. medium-btn-text, small-text or your variants.

File: src/app/Page.module.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import '@/theme/mixins';
+
+.one {
```
Comment (OlhaPo) [inline src/app/Page.module.scss:3]: Maybe such class names like one, two, three etc might be confusing, better to use more descriptive and meaningful names. My suggestion: heading-primary, heading-secondary, subtitle, card-title, accent-text, main-text, big-btn-text. medium-btn-text, small-text or your variants.

Comment (biryukov-m) [reply src/app/Page.module.scss]: There were examples for teammates of using mixins, we will get rid of them very soon, renamed anyway

**Comment by biryukov-m** [comment]
fixed

**Review by OlhaPo — APPROVED** [review]

**Pull request merged**